src/components/primary-button/PrimaryButton.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import Button, { ButtonProps } from '../button';
+
+const PrimaryButton = ({ children, className = '', ...rest }: Partial<ButtonProps>) => (
```

Comment:
Partial makes all the props optional, right? Is it needed here?

Reply:
Without partial, TS errors arise when doing `<PrimaryButton className="foo" >` because it complains about missing "type" (and everything else) on it, which is a required prop of "Button". Even though within Button its part of defaultProps, I am guessing typescript cannot infer that from the view point of PrimaryButton.